docs/spdx-tag-value-mapping.md

```diff
@@ -0,0 +1,151 @@
+SPDX-License-Identifier: BSD-2-Clause
+
+# SPDX Mapping for Tern
+
+This file describes a potential mapping between Tern's data structures and the relevant fields of an [SPDX](https://spdx.org/) tag-value document. This is intended to assist with enabling Tern to output its findings in SPDX tag-value format, for consumption by other tools.
+
+In particular, it will focus on highlighting any necessary data items that are deemed "mandatory" by the SPDX specification.
+
+Version 2.1 of [the SPDX specification](https://spdx.org/specifications) will be used, since it is the most recent minor release. Relevant details of the SPDX tag-value format can be found in [the overview document](spdx-tag-value-overview.md) in this directory.
+
+## Relevant Fields
+
+The following table contains (1) all fields that are designated by the SPDX specification as 'mandatory', for the SPDX elements that Tern is likely to use; and (2) any additional fields that are optional but likely to be useful for Tern.
+
+Thoughts on mapping these fields to Tern's data model are described in the **Mapping** section below.
+
+Section (§) numbers are references to the relevant portions of [the SPDX specification](https://spdx.org/specifications), version 2.1. Examples should be assumed to be on a single line, though a Markdown formatter might split them across lines.
+
+### Document Creation
+
+The following fields should appear *once*, at the beginning of the SPDX document:
+
+ §  | SPDX Field name | Mandatory? | Brief description    | Example
+----|-----------------|------------|----------------------|---------
+2.1 | SPDX Version    |    Yes     | version of SPDX spec | `SPDXVersion: SPDX-2.1`
+2.2 | Data License    |    Yes     | license for SPDX metadata in the document itself; always `CC0-1.0` | `DataLicense: CC0-1.0`
+2.3 | SPDX Identifier |    Yes     | identifier for the SPDX document itself; always `SPDXRef-DOCUMENT` | `SPDXID: SPDXRef-DOCUMENT`
+2.4 | Document Name   |    Yes     | human-readable name for the SPDX document itself | `DocumentName: Tern report for ACME Dockerfile`
+2.5 | SPDX Document Namespace | Yes| unique absolute URI for the SPDX document itself | `DocumentNamespace: https://example.com/spdxdocs/tern-report-ACME-1.0.1-123456`
```

Comment:
Tern doesn't host documents. So this should be manually updated once completed. The spec says one  can use `https://spdx.org/spdxdocs` if they're not hosting the document.